src/components/Button.js

```diff
@@ -0,0 +1,9 @@
+import { render } from "../shared/dom";
+
+const template = (text) => `
+    <button class='btn>${text}</button>
+`;
+
+export const Button = ({renderOn, text}) => {
```

Comment:
It's my bad, poor task description, I think that we need two more arguments in here
1. Function that we will hoop up to this button, so it would actually do something
2. Button type, for example we could pass the "submit" if we would use it in form
3. Extra class name, if we want to change some styling in button based on needs